### ISSUE TITLE
SampleHandler failure to find files doesn't offer any info

### ISSUE THREAD

**Issue by rsmith54** [rsmith54/RJigsawTools#101]
@mattleblanc 's issue is what happens when the SampleHandler object is empty.  This is actually quite annoying and should have some higher verbosity (probably not by default since SampleHandler spits out a lot of stuff with sh.printContent() ).

I made a few quick changes to look at this, but probably this is just some annoying failure in fillSampleHandler (maybe SampleHandler has changed?) .

This goes along with whatever was going on with FAX yesterday, but that's not our fault .

**Comment by lawrenceleejr** [comment]
We could add a check in the common python or something that checks that the
final sample handler object isn't empty.

**Comment by rsmith54** [comment]
Yeah, I have that coded right now.  I guess the thing is maybe we should
update the SH scanning to be not as dumb as it is by default with the newer
ScanDir vs scanDir.
